README.md

```diff
@@ -42,5 +42,5 @@ To ensure that existing features still work as expected after your changes, you
 
 When the application is ready:
 
-- Run `npm run cypress:run` in the `core/examples/luigi-sample-angular` folder to start tests in the headless browser.
-- Run `npm run cypress:open` in the `core/examples/luigi-sample-angular` folder to start tests in the interactive mode.
\ No newline at end of file
+- Run `npm run e2e:run` in the __core/examples/luigi-sample-angular__ folder to start tests in the headless browser.
```

Comment:
Why have you introduced "_" ? According to our guidelines, we format paths with code, so it should be: `core/examples/luigi-sample-angular` as in the previous version. Or am I missing something?

Reply:
Actually I am not sure, since this was in the master and I had code stuff, but saw underscores coming from master. Then I'll change it back to code, thanks